wodles/aws/tests/test_server_access.py

```diff
@@ -179,9 +179,9 @@ def test_aws_server_access_check_bucket_handles_exceptions_when_empty_bucket(moc
 
 
 @pytest.mark.parametrize('error_code, exit_code', [
-    (aws_bucket.THROTTLING_EXCEPTION_ERROR_CODE, utils.THROTTLING_ERROR_CODE),
-    (aws_bucket.INVALID_CREDENTIALS_ERROR_CODE, utils.INVALID_CREDENTIALS_ERROR_CODE),
-    (aws_bucket.INVALID_REQUEST_TIME_ERROR_CODE, utils.INVALID_REQUEST_TIME_ERROR_CODE),
+    (constants.THROTTLING_EXCEPTION_ERROR_CODE, utils.THROTTLING_ERROR_CODE),
```

Comment:
I would recommend 2 things here:
- Move constants from the utils files to unify them into the constant file. 
- Since we are doing minor package improvements rename the `aws_utils` into `utils` and also fix the imports for it.

Reply:
When changing `aws_utils.py` to `utils.py` the following error appeared
```
aws/tests/test_aws_bucket.py::test_aws_bucket_initializes_properly[None] - AttributeError: <module 'utils' from '/Users/framos/Documents/Wazuh/Repositories/wazuh/wodles/aws/tests/./utils.py'> does not have the attribute 'get_wazuh_version'
```

This is because all patches import using the same file name for the wodles utils, causing a conflict

```
patch('wazuh_integration.utils.find_wazuh_path', return_value=constants.TEST_WAZUH_PATH)
```

I think it would be appropriate to leave the file name as is.